salt/metalk8s/kubernetes/cni/calico/configured.sls

```diff
@@ -1,8 +1,12 @@
+{%- from "metalk8s/map.jinja" import certificates with context %}
 {%- from "metalk8s/map.jinja" import kube_api with context %}
 {%- from "metalk8s/map.jinja" import kubernetes with context %}
 {%- from "metalk8s/map.jinja" import networks with context %}
 
 {%- set kubernetes_service_ip = salt.metalk8s_network.get_kubernetes_service_ip() %}
+{%- set cn = salt['slsutil.renderer'](
+    string=certificates.kubeconfig.calico.cn
+) %}
```

Comment:
This is really needed ? From what I remember pillar already get jinja rendered (but maybe it's only the case when you retrieve it from pillar directly and not in `map.jinja` :thinking: )

Reply:
You're right, just re tested. I don't know what I did but during a test it was not rendered, I may have failed something something. :)